src/GraphQl/Subscription/MercureSubscriptionUriGeneratorInterface.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\GraphQl\Subscription;
+
+/**
+ * Generates Mercure-related URIs from a subscription ID.
+ *
+ * @experimental
+ *
+ * @author Alan Poulain <contact@alanpoulain.eu>
+ */
+interface MercureSubscriptionUriGeneratorInterface
```

Comment:
`MercureSubscriptionIriGeneratorInterface` for consistency (we use IRI everywhere).